services/shuttle-rocket/src/lib.rs

```diff
@@ -18,11 +18,18 @@
 //! }
 //! # }
 //! ```
+use rocket::http::Status;
+use rocket::response::status;
 use std::net::SocketAddr;
 
 /// A wrapper type for [rocket::Rocket<rocket::Build>] so we can implement [shuttle_runtime::Service] for it.
 pub struct RocketService(pub rocket::Rocket<rocket::Build>);
 
+#[rocket::get("/healthz")]
+fn health_check() -> status::Custom<()> {
+    status::Custom(Status::Ok, ())
+}
```

Comment:
Nit: could we just return an empty tuple here? We'll just be interested in the response status, which I believe will default to 200 OK

Reply:
yeah we could that like this
```rust
#[rocket::get("/healthz")]
fn health_check() {}
```
and it does work wouldn't it better to explicitly show that we are responding with `Status:Ok`

Reply:
This is only a nit. Feel free to do it as you wish. I prefer the explicit aspect, but I have no strong opinion about it.